lib/tasks/update.rake

```diff
@@ -78,4 +78,16 @@ namespace :update do
       DatabaseUpdateWorker.perform_in(1.minutes, { county_id: c[0], case_number: c[1] })
     end
   end
+
+  desc 'Set the is_error flag on court_cases'
+  task is_error: [:environment] do
+    associations = [:parties, :current_judge, :counsels, :counts, :events, :docket_events]
+    bar = ProgressBar.new(CourtCase.all.count)
+    CourtCase.in_batches(of: 1000) do |court_cases|
+      court_cases.includes(associations).each do |court_case|
+        court_case.update!(is_error: true) if court_case.error?
+      end
+      bar.increment!(1000)
+    end
+  end
```

Comment:
I put this here instead of as a one-off in case we need to re-run because error requirements change? 
Also there's a couple ways we could auto run these on deploy as part of migrations/data-migrations for full automation

Reply:
Running this currently takes about 18 minutes locally and there is a similar count on production